Makefile

```diff
@@ -208,11 +208,22 @@ prepare: prepare-build
 prepare-help: prepare-build
 	mono --debug $(PREPARE_EXE) -h
 
+.PHONY: shutdown-compiler-server
+shutdown-compiler-server:
+	# Ensure the VBCSCompiler.exe process isn't running during the mono update
+	pgrep -lfi VBCSCompiler.exe 2>/dev/null || true
+	$(eval pid=$(shell sh -c "pgrep -lfi VBCSCompiler.exe 2>/dev/null" | awk '{ print $$1 }'))
```

Comment:
This line needs to end with a `\` in order for `$$pid` to be usable on the following line.

(By default, each line in a Makefile gets its own shell.)

I also don't understand why this. is using `$(eval)` as well.

Reply:
See comment below. Addressed with commit 8a3d64455a394724a94b78a28b9a035897ac8bfb